src/app_engine/constants.py

```diff
@@ -74,6 +74,9 @@
 PARAM_MESSAGE = 'message'
 PARAM_ROOM_ID = 'roomId'
 PARAM_METADATA = 'metadata'
+PARAM_TYPE = 'type'
+
+TYPE_RINGING = 'ringing'
```

Comment:
If we go with /ack, I think the 'type' should be the GCM message type that is ack'd.